CI Visibility - Swift - Update instructions (#15677)
DataDog/documentation: content/en/continuous_integration/tests/swift.md

3. If you run UITests, also link the app running the tests with this library.

### Using Swift Package Project

> 1. Add `dd-sdk-swift-testing` to your package dependencies array, eg:

Reviewer: ```suggestion
1. Add `dd-sdk-swift-testing` to your package dependencies array, for example:
```

{{< code-block lang="swift" >}}
.package(url: "https://github.com/DataDog/dd-sdk-swift-testing.git", from: "2.1.0")
{{< /code-block >}}

> 2. Add the the testing framework to your testing targets dependencies. Add the following line to your test targets dependencies array:

Reviewer: ```suggestion
2. To add the testing framework to your testing targets' dependencies, add the following line to your test targets dependencies array:
```

**Examples**: `ci`, `local`

`SRCROOT`

> : The path to the project location. Use `$(SRCROOT)` for the value if using Xcode, because it is automatically set by it.<br/>

Reviewer: ```suggestion
: The path to the project location. If using Xcode, use `$(SRCROOT)` for the value, because it is automatically set by it.<br/>
```
